src/connectors/cisco_umbrella.py

```diff
@@ -33,7 +33,7 @@
         'name': 'organization_id',
         'title': "Cisco Umbrella Organization Id",
         'prompt': "Your Cisco Umbrella Organization Id",
-        'type': 'str',
+        'type': 'int',
```

Comment:
going to add this to the general runner here as we do with JSON: https://github.com/chanzuckerberg/SnowAlert/blob/v1.8.6/src/runners/connectors_runner.py#L43-L44